org.adempiere.base/src/org/compiere/model/MSysConfig.java

```diff
@@ -128,6 +128,7 @@ public class MSysConfig extends X_AD_SysConfig
     public static final String LOCATION_MAX_CITY_ROWS = "LOCATION_MAX_CITY_ROWS";
     public static final String LOGIN_HELP_URL = "LOGIN_HELP_URL";
 	public static final String LOGIN_PREFIX_SEPARATOR = "LOGIN_PREFIX_SEPARATOR";
+    public static final String LOGIN_SELECT_ROLE_HELP_URL = "LOGIN_SELECTION_HELP_URL";

```

Comment:
hi @CarlosRuiz-globalqss , you forgot to change the text.

Reply:
> hi @CarlosRuiz-globalqss , you forgot to change the text.

Thanks!   Fixed